pvanalytics/quality/gaps.py

```diff
@@ -335,7 +335,7 @@ def complete(series, minimum_completeness=0.333, freq=None):
 
     See Also
     --------
-    :py:func:`completeness_score`
+    completeness_score
```

Comment:
No need for the `:py:func:` business in "See Also"; sphinx links them automatically anyway.